pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TopicsConsumerImplTest.java

```diff
@@ -685,92 +685,6 @@ public void testMultiTopicsMessageListener() throws Exception {
         consumer.close();
     }
 
-
-    /**
-     * Test topic partitions auto subscribed.
-     *
-     * Steps:
-     * 1. Create a consumer with 2 topics, and each topic has 2 partitions: xx-partition-0, xx-partition-1.
-     * 2. produce message to xx-partition-2, and verify consumer could not receive message.
```

Comment:
How about just remove this step-2 instead of the whole test case?

Reply:
sure, updated to keep test case to still test can auto subscribe newly added partitions.